Patches/Thrumkin/Patch_Race_Thrumkin.xml

```diff
@@ -1,134 +1,168 @@
+		<mods>
+			<li>[SYR] Thrumkin</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- Combat -->
+				<li Class="PatchOperationAddModExtension">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Thrumkin"]</xpath>
+					<value>
+						<li Class="CombatExtended.RacePropertiesExtensionCE">
+							<bodyShape>Humanoid</bodyShape>
 						</li>
-						<li Class="CombatExtended.ToolCE">
-							<label>horn</label>
-							<capacities>
-								<li>Stab</li>
-							</capacities>
-							<power>21</power>
-							<cooldownTime>2.39</cooldownTime>
-							<linkedBodyPartsGroup>HornAttackTool</linkedBodyPartsGroup>
-							<chanceFactor>0.4</chanceFactor>
-							<armorPenetrationSharp>2.45</armorPenetrationSharp>
-							<armorPenetrationBlunt>3.669</armorPenetrationBlunt>
+					</value>
+				</li>
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Thrumkin"]/comps</xpath>
+					<value>
+						<li>
+							<compClass>CombatExtended.CompPawnGizmo</compClass>
 						</li>
-						<li Class="CombatExtended.ToolCE">
-							<label>left fist</label>
-							<capacities>
-								<li>Blunt</li>
-							</capacities>
-							<power>2</power>
-							<cooldownTime>1.12</cooldownTime>
-							<linkedBodyPartsGroup>LeftHand</linkedBodyPartsGroup>
-							<surpriseAttack>
-							<extraMeleeDamages>
-								<li>
-									<def>Stun</def>
-									<amount>20</amount>
-								</li>
-							</extraMeleeDamages>
-							</surpriseAttack>
-							<armorPenetrationBlunt>0.528</armorPenetrationBlunt>
-						</li>
-						<li Class="CombatExtended.ToolCE">
-							<label>right fist</label>
-							<capacities>
-								<li>Blunt</li>
-							</capacities>
-							<power>2</power>
-							<cooldownTime>1.12</cooldownTime>
-							<linkedBodyPartsGroup>RightHand</linkedBodyPartsGroup>
-							<surpriseAttack>
+						<li Class="CombatExtended.CompProperties_Suppressable" />
+					</value>
+				</li>
+				<!-- thrumkin rebalances -->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName = "Thrumkin"]/statBases</xpath>
+					<value>
+						<!-- combat stats -->
+						<AimingAccuracy>0.9</AimingAccuracy>
+						<MeleeCritChance>1.34</MeleeCritChance>
+						<MeleeParryChance>1.27</MeleeParryChance>
+						<Suppressability>0.8</Suppressability>
+						<ReloadSpeed>0.9</ReloadSpeed>
+						<MeleeDodgeChance>1.06</MeleeDodgeChance>
+						<!-- other stats -->
+						<CarryWeight>50</CarryWeight>
+					</value>
+				</li>
+				<!-- should've modified the values sooner -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName = "Thrumkin"]/statBases/Mass</xpath>
+					<value>
+						<Mass>120</Mass>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/AlienRace.ThingDef_AlienRace[defName = "Thrumkin"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>2</ArmorRating_Sharp>
```

Comment:
The issue with the additional armor, especially this much, is that it makes them totally impervious to most melee attacks, including most attacks from other Thrumkins. Perhaps you could give them less armor (like, 0.1 - 0.3 sharp) and boost HP or something?